[2024-01-27T18:02:38.161+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing_dag.store_user manual__2024-01-27T18:02:31.032462+00:00 [queued]>
[2024-01-27T18:02:38.168+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing_dag.store_user manual__2024-01-27T18:02:31.032462+00:00 [queued]>
[2024-01-27T18:02:38.170+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-01-27T18:02:38.171+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-01-27T18:02:38.172+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-01-27T18:02:38.181+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2024-01-27 18:02:31.032462+00:00
[2024-01-27T18:02:38.186+0000] {standard_task_runner.py:55} INFO - Started process 8106 to run task
[2024-01-27T18:02:38.189+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing_dag', 'store_user', 'manual__2024-01-27T18:02:31.032462+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/user_processing_dag.py', '--cfg-path', '/tmp/tmpw2_frksa']
[2024-01-27T18:02:38.191+0000] {standard_task_runner.py:83} INFO - Job 45: Subtask store_user
[2024-01-27T18:02:38.240+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing_dag.store_user manual__2024-01-27T18:02:31.032462+00:00 [running]> on host 0c5b25b371b7
[2024-01-27T18:02:38.283+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing_dag
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2024-01-27T18:02:31.032462+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-27T18:02:31.032462+00:00
[2024-01-27T18:02:38.285+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2024-01-27T18:02:38.294+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-01-27T18:02:38.301+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-01-27T18:02:38.310+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing_dag, task_id=store_user, execution_date=20240127T180231, start_date=20240127T180238, end_date=20240127T180238
[2024-01-27T18:02:38.322+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-01-27T18:02:38.337+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
